src/spec/vk.xml

```diff
@@ -977,7 +977,7 @@ private version is maintained in the 1.0 branch of the member gitlab server.
             <member><type>VkSampleCountFlagBits</type>  <name>rasterizationSamples</name><comment>Number of samples used for rasterization</comment></member>
             <member><type>VkBool32</type>               <name>sampleShadingEnable</name><comment>optional (GL45)</comment></member>
             <member><type>float</type>                  <name>minSampleShading</name><comment>optional (GL45)</comment></member>
-            <member optional="true" len="latexmath:[\lceil{\mathit{rasterizationSamples} \over 32}\rceil]" altlen="ceil(rasterizationSamples / 32)">const <type>VkSampleMask</type>*    <name>pSampleMask</name><comment>Array of sampleMask words</comment></member>
+            <member optional="true" len="latexmath:[\lceil{\mathit{rasterizationSamples} \over 32}\rceil]" altlen="rasterizationSamples / 64 + 1">const <type>VkSampleMask</type>*    <name>pSampleMask</name><comment>Array of sampleMask words</comment></member>
```

Comment:
How is "ceil(rasterizationSamples/32)" equivalent to "rasterizationSamples/64 + 1"?

Reply:
Well, `rasterizationSamples` is 2<sup>*N*</sup> where 0&le;*N*&le;6, and so it works for that range
I am being too smartistic here, and so it is not future proof.

`(rasterizationSamples + 31)/32` then should work for all *N*&ge;0, well for any integer 0&le;*rasterizationSamples*&le;(`UINT32_MAX - 31`) in fact